src/icon-manifest/supportedExtensions.ts

```diff
@@ -14,7 +14,7 @@ export const extensions: IFileCollection = {
     { icon: 'actionscript', extensions: [], languages: [languages.actionscript], format: FileFormat.svg },
     { icon: 'ai', extensions: ['ai'], format: FileFormat.svg },
     { icon: 'ai2', extensions: ['ai'], format: FileFormat.svg, disabled: true },
-    { icon: 'angular', extensions: ['angular-cli.json'], filename: true, format: FileFormat.svg },
+    { icon: 'angular', extensions: ['.angular-cli.json'], filename: true, format: FileFormat.svg },
```

Comment:
Hi @ulic75, please keep both formats for the moment. So people who may be using old version still get the icon.